src/engine/enginebuffer.cpp

```diff
@@ -1179,7 +1179,7 @@ void EngineBuffer::processSeek(bool paused) {
             return;
     }
 
-    if ((seekType & SEEK_PHASE) && !paused && m_pQuantize->toBool()) {
+    if (!paused && ((seekType & SEEK_PHASE) || m_pQuantize->toBool())) {
```

Comment:
I am not familiar with this area of the code. Can you clarify what this change does?

Reply:
This allows to sync phase independent from quantize now. The sync signal of the controls that must not sync phase if quatize is of, is done in their buissnes logic.

Reply:
Okay thank you for explaining.